### PR TITLE
Update bitlocker-device-encryption-overview-windows-10.md

### PR DESCRIPTION
**Pull request by mardahl**
Added information about deprecation that is VERY important to point out, and removed links to get MBAM.

Added new links and relevant information.

#ATCP

### CODE REVIEW

File: windows/security/information-protection/bitlocker/bitlocker-device-encryption-overview-windows-10.md [hunk 1]
```diff
@@ -144,4 +144,9 @@ Part of the Microsoft Desktop Optimization Pack, MBAM makes it easier to manage
 * Offers an IT-customizable recovery user experience.
 * Supports Windows 10.
 
-For more information about MBAM, including how to obtain it, see [Microsoft BitLocker Administration and Monitoring](/microsoft-desktop-optimization-pack/) on the MDOP TechCenter.
+> [!IMPORTANT]
+>Enterprises can use Microsoft BitLocker Administration and Monitoring (MBAM) to manage client computers with BitLocker that are domain-joined on-premises until mainstream support ends in July 2019 or they can receive extended support until April 2026
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-device-encryption-overview-windows-10.md:148]:
```suggestion
> Enterprises could use MBAM to manage client computers with BitLocker that are domain-joined on-premises until mainstream support ended in July 2019, or they could receive extended support until April 2026.
```

File: windows/security/information-protection/bitlocker/bitlocker-device-encryption-overview-windows-10.md [hunk 2]
```diff
@@ -144,4 +144,9 @@ Part of the Microsoft Desktop Optimization Pack, MBAM makes it easier to manage
...
+
+Going forward, the functionality of MBAM will be incorporated into Microsoft Endpoint Manager Config Manager (MEMCM), see [Features in Configuration Manager technical preview version 1909](/mem/configmgr/core/get-started/2019/technical-preview-1909#bkmk_bitlocker)
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-device-encryption-overview-windows-10.md:150]:
```suggestion
Going forward, the functionality of MBAM will be incorporated into Configuration Manager. For more details, see [Features in Configuration Manager technical preview version 1909](/mem/configmgr/core/get-started/2019/technical-preview-1909#bkmk_bitlocker).
```

File: windows/security/information-protection/bitlocker/bitlocker-device-encryption-overview-windows-10.md [hunk 3]
```diff
@@ -144,4 +144,9 @@ Part of the Microsoft Desktop Optimization Pack, MBAM makes it easier to manage
...
+
+Customers not using Config Manager can utilize the built-in features of Azure AD and Microsoft Endpoint Manager Microsoft Intune (MEMMI) for administration and monitoring, see [Monitor device encryption with Intune](/mem/intune/protect/encryption-monitor)
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-device-encryption-overview-windows-10.md:152]:
```suggestion
If you do not use Configuration Manager, you could use the built-in features of Azure AD and Microsoft Intune in Microsoft Endpoint Manager for administration and monitoring. For more details, see [Monitor device encryption with Intune](/mem/intune/protect/encryption-monitor).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mardahl, kindly commit the suggested changes. Many thanks.

**Comment by mardahl** [comment]
Thank you for the corrections. I have committed them :)

**Review by denisebmsft — APPROVED** [review]
Thank you @mardahl @JohanFreelancer9

**Pull request merged**